### ISSUE TITLE
post-test-infra-push-prow: wrong bazel version

### ISSUE THREAD

**Issue by stevekuznetsov** [kubernetes/test-infra#13013]
**What happened**:

Bazel version was not upgraded in `post-test-infra-push-prow`.

**Please provide links to example occurrences, if any**:

https://prow.k8s.io/job-history/kubernetes-jenkins/logs/post-test-infra-push-prow

**Comment by stevekuznetsov** [comment]
I can't fix this one myself :(

```
$ images/gcloud-bazel/push.sh 
Cloning into 'rules_k8s'...
remote: Enumerating objects: 1541, done.
remote: Total 1541 (delta 0), reused 0 (delta 0), pack-reused 1541
Receiving objects: 100% (1541/1541), 418.50 KiB | 1.44 MiB/s, done.
Resolving deltas: 100% (922/922), done.
make: Entering directory '/home/skuznets/fakego/src/k8s.io/test-infra/images/gcloud-bazel/rules_k8s/images/gcloud-bazel'
gcloud builds submit \
	--config=cloudbuild.yaml \
	--substitutions=_IMG=gcr.io/k8s-testimages/gcloud-bazel,_USER=skuznets,SHORT_SHA=v20190613-v0.1-87-g2e2d1e5 \
	--project=k8s-testimages \
	.
ERROR: (gcloud.builds.submit) User [skuznets@redhat.com] does not have permission to access b [k8s-testimages_cloudbuild] (or it may not exist): skuznets@redhat.com does not have serviceusage.services.use access to project 597082399059.
make: *** [Makefile:24: push] Error 1
make: Leaving directory '/home/skuznets/fakego/src/k8s.io/test-infra/images/gcloud-bazel/rules_k8s/images/gcloud-bazel'
```

**Comment by Katharine** [comment]
Fixed by #13015 - it was using the wrong image.
